h11/_receivebuffer.py

```diff
@@ -16,33 +16,10 @@
 #   of constantly copying
 # WARNING:
 # - I haven't benchmarked or profiled any of this yet.
-#
-# Note that starting in Python 3.4, deleting the initial n bytes from a
-# bytearray is amortized O(n), thanks to some excellent work by Antoine
-# Martin:
-#
-#     https://bugs.python.org/issue19087
```

Comment:
I think it'd be helpful to keep the paragraph above.

Reply:
Fair point, yes. I've addressed it as an inline comment.